src/data_augmentation.cpp

```diff
@@ -37,4 +37,29 @@ void DataAugmentation::AverageFilter(const int &kernel){
   cv::namedWindow("Average Filter",CV_WINDOW_NORMAL); 
   cv::imshow("Average Filter",out_);
   cv::imwrite("../../imgs/average.png",out_);
+}
+
+void DataAugmentation::SaltPepper(const float percentage){
+  cv::Mat pepperized_ = in_.clone();
+  int total = percentage * pepperized_.cols * pepperized_.rows;
+  for(int i = 0; i<total; i++){
+    //Choose a random pixel between 0 - (im.cols-1)
```

Comment:
modifica comentario a que indique el nombre de la matriz que estas usando. 
Exemplo: between 0 and (out_.cols-1)

Reply:
Done